mubench.reviewsite/tests/CSVHelperTest.php

```diff
@@ -0,0 +1,366 @@
+<?php
+
+use Monolog\Logger;
+use PHPUnit\Framework\TestCase;
+use MuBench\ReviewSite\Model\ExperimentResult;
+use MuBench\ReviewSite\Model\DetectorResult;
+use MuBench\ReviewSite\Model\Detector;
+use MuBench\ReviewSite\Model\Misuse;
+use MuBench\ReviewSite\Model\Review;
+
+use \MuBench\ReviewSite\CSVHelper;
+
+class CSVHelperTest extends TestCase
+{
+
+    private $csv_helper;
+
+    function setUp()
+    {
+        $this->csv_helper = new CSVHelper();
+    }
+
+    public function test_csv_from_stats_ex1()
+    {
+        $stats = [];
+        $stats[1] = new DetectorResult(
+            new Detector("-d1-", 1)
```

Comment:
Please extract constants as the `detector_id` here into a local variable (const?) to make their reuse obvious. Btw. you create two detectors with different name, but same id here. You could also extract the detectors into variables and use `$detector->getId()` to reference them.